data/presets/_pipeline.json

```diff
@@ -0,0 +1,18 @@
+{
+    "icon": "iD-pipeline-line",
+    "fields": [
+        "pipeline",
+        "operator"
+    ],
+    "geometry": [
+        "point",
+        "vertex",
+        "line",
+        "area"
+    ],
+    "tags": {
+        "natural": "*"
```

Comment:
I assume this should have been the `pipeline` tag here, right? I've totally overlooked it when reviewing the PR. :see_no_evil:

Reply:
This was spotted in https://www.transifex.com/openstreetmap/id-editor/translate/#pl/$/442611093 by @gdabski

It's fixed in 4f70a708b3a96c02104777f1cfecef895c73970e